src/summary.js

```diff
@@ -0,0 +1,26 @@
+function summarize(news) {
```

Comment:
again, maybe this function is too big? Could be split into a api and then format funciton?